docs/articles/blog/2016-12-8-testcafe-v0-11-0-released.md

```diff
@@ -0,0 +1,223 @@
+
+```js
+const selector = Selector(arg => /* selector code */);
+const selector2 = selector('someArg').find('span'); // <-- doesn't throw anymore
+
+test ('Some test', async t=> {
+...
+});
+```
+
+However, once you'll try to obtain element property outside of test context it will throw:
+
+```js
+const selector = Selector(arg => /* selector code */);
+
+async getId() {
+  return await selector('someArg').id; // throws
+}
+
+getId();
+
+test ('Some test', async t=> {
+...
+});
+```
+
+#### Index filter is not ignored anymore if selector returns single node
+
+Previously if selector returned single node `index` was ignored:
+
+```js
+Selector('#someId', { index: 2 } ); // index is ignored and selector returns element with id `someId`
+```
+
+however it's not a case now:
+
+```js
+Selector('#someId').nth(2); // returns `null`, since there is only one element in matching set with id `someId`
+```
+
+#### Deprecated API
+
+* [`t.select` method](http://devexpress.github.io/testcafe/documentation/test-api/selecting-page-elements/selectors.html#one-time-selection) - use `Selector` instead:
+
+```js
+const id = await t.select('.someClass').id;
+
+// can be replaced with
+
+const id = await Selector('.someClass').id;
+```
+
+* [selectorOptions.index](http://devexpress.github.io/testcafe/documentation/test-api/selecting-page-elements/selector-options.html#optionsindex) - use [selector.nth()](http://devexpress.github.io/testcafe/documentation/test-api/selecting-page-elements/selectors.html#nth) instead.
+* [selectorOptions.text](http://devexpress.github.io/testcafe/documentation/test-api/selecting-page-elements/selector-options.html#optionstext) - use [selector.withText()](http://devexpress.github.io/testcafe/documentation/test-api/selecting-page-elements/selectors.html#withtext) instead.
+* [selectorOptions.dependencies](http://devexpress.github.io/testcafe/documentation/test-api/selecting-page-elements/selector-options.html#optionsdependencies) - use [filtering](https://devexpress.github.io/testcafe/documentation/test-api/selecting-page-elements/selectors.html#filter-multiple-dom-nodes) and [hierarchical](https://devexpress.github.io/testcafe/documentation/test-api/selecting-page-elements/selectors.html#find-elements-by-dom-hierarchy) methods to build combined selectors instead.
+
+### ⚙ Built-in assertions. ([#998](https://github.com/DevExpress/testcafe/issues/998))
+
+TestCafe now ships with [numerous built-in BDD-style assertions](http://devexpress.github.io/testcafe/documentation/test-api/assertions/assertion-api.html).
+If the TestCafe assertion receives a [Selector's property](https://devexpress.github.io/testcafe/documentation/test-api/selecting-page-elements/dom-node-state.html#members-common-across-all-nodes) as an actual value, TestCafe uses the [smart assertion query mechanism](http://devexpress.github.io/testcafe/documentation/test-api/assertions/index.html#smart-assertion-query-mechanism):
```

Comment:
We gave [Selector DOM node state property](../selecting-page-elements/selectors.md#define-assertion-actual-value) link in this sentence in assertion documentation, let's chose one variant for this